lib/wor/paginate/adapters/iterable.rb

```diff
@@ -11,8 +11,8 @@ def adapt?(content)
         def adapt(content, page, limit)
           content_array = content.to_a
           sliced_content = content_array.slice((page - 1) * limit, limit)
-          Wor::Paginate::Formatter.format(sliced_content, sliced_content.count,
-                                          content_array.count, page)
+          Wor::Paginate::Config.formatter.format(sliced_content, sliced_content.count,
```

Comment:
Should we make it so that the user can send a custom formatter per request as well?